install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -11,18 +11,20 @@ metadata:
     release: {{ .Release.Name }}
 data:
   mesh: |-
+    {{- if .Values.mixer.policy.enabled }}
     # Set the following variable to true to disable policy checks by the Mixer.
     # Note that metrics will still be reported to the Mixer.
     disablePolicyChecks: {{ .Values.global.disablePolicyChecks }}
```

Comment:
Remove the disablePolicyChecks thing from the global stuff. Move this into the if block below.
and set disablePolicyChecks : {{ ! .Values.mixer.policy.enabled }} or some such way..